N: 720

max_speed: 0.35
min_speed: 0.0
max_yawrate: 1.0
max_accel: 3.0
max_dyawrate: 1.5         
 
v_reso: 0.1
yawrate_reso: 0.1

dt: 0.1
predict_time: 3.0

to_goal_cost_gain: 3.0
speed_cost_gain: 0.1
obstacle_cost_gain: 0.01

robot_radius: 0.25

roomba_v_gain: 0.4
roomba_omega_gain: 0.5
                        
